std-bits/base/src/main/java/org/enso/base/math/Math_Utils.java

```diff
@@ -0,0 +1,119 @@
+package org.enso.base.math;
+
+public class Math_Utils {
```

Comment:
Once we have this, I assume we should be able to also get vectorized implementations for columns at basically no additional cost.

Reply:
I'll implement that in this PR.